### ISSUE TITLE
Buttons Missing Titles

### ISSUE THREAD

**Issue by aamct2** [storybooks/storybook#6123]
**Describe the bug**

A number of the core interface buttons that are represented with icons do not specify titles or some other attribute that would be used by screen readers. This makes the experience less accessible.

Specifically:
- [x] The ellipsis button at the top of the sidebar that displays the shortcuts
- [x] The clear button that removes all typed text in the search bar (in the sidebar)
- [x] The open canvas in a new tab/window button in the upper-right corner

**To Reproduce**
1. Load Storybook
1. Inspect any of the above button elements
1. No title attribute is set

Alternatively, run an automated accessibility checker (such as Lighthouse in the Chrome Developer Tools) to see them picked up.

**Expected behavior**

All buttons should have a title or similar attribute available for screen readers.

**System:**
 - OS: macOS 10.14.3
 - Browser: Chrome 73.0.3683.75
 - Version: `next` branch as of Commit b45e4a176f33f869a39dc1635ca9ee9b7081fcad

**Comment by aamct2** [comment]
Resolved in #6124

**Comment by shilman** [comment]
w00t!! I just released https://github.com/storybooks/storybook/releases/tag/v5.1.0-alpha.8 containing PR #6124 that references this issue. Upgrade today to try it out!

Because it's a pre-release you can find it on the `@next` NPM tag.

**Comment by shilman** [comment]
Zoinks!! I just released https://github.com/storybooks/storybook/releases/tag/v5.0.3 containing PR #6124 that references this issue. Upgrade today to try it out!

**Comment by shilman** [comment]
ZOMG!! I just released https://github.com/storybooks/storybook/releases/tag/v5.0.4 containing PR #6124 that references this issue. Upgrade today to try it out!
